app/mautic_logging/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.
```

Comment:
We can test the model with some mock responses, or rather update tests related to the mautic_proxy